aldryn_reversion/admin.py

```diff
@@ -120,8 +126,19 @@ def log_addition(self, request, obj):
             "Initial version of %(object_repr)s.%(translation_info)s") % {
                 'object_repr': build_obj_repr(obj),
                 'translation_info': get_translation_info_message(obj)}
+        objects = [obj]
+        # django-reversion >1.9.0 no longer has get_revision_instances
+        # but for supporting older versions we will check for it and use
+        if hasattr(self, 'get_revision_instances'):
+            for additional_object in self.get_revision_instances(
+                    request, obj):
+                if additional_object not in objects:
+                    objects.append(additional_object)
+        # previous implementation was to use self.get_revision_data
+        # but that was also removed in 1.9.0 since it was a duplicate of logic
+        # that is already present in save_revision or its related calls.
         self.revision_manager.save_revision(
-            self.get_revision_data(request, obj),
+            objects,
```

Comment:
``get_revision_instances`` was a place to add additional objects to revision, you could override that method on admin class and have benefits from that, in 1.9.0 however things were changed, and  ``get_revision_instances`` is no longer there, but in case it was used somewhere previously i've added a call to it.
Now, the other thing is  ``get_revision_data`` which also was removed starting from 1.9.0, i assume that that was another place to add custom things if you wanted to modify the way serialization is done. 
This ``get_revision_data`` was a duplicate of ``revision_manager.save_revision`` first steps https://github.com/etianen/django-reversion/blob/450808334ecefc68c4c98bc337bb3d6128eb74ff/src/reversion/revisions.py#L444 and compared to https://github.com/etianen/django-reversion/blob/450808334ecefc68c4c98bc337bb3d6128eb74ff/src/reversion/admin.py#L151 which has only one difference - a call to ``get_revision_instances`` to get objects.

Actually this whole ``get_revision_instances`` can be dropped. I think aldryn addons do not use that, i've only encountered this in ``django-cms`` admin classes for pages. And maybe it is not a very good idea to support deprecated functions on our own.